web/libs/editor/src/mixins/Required.js

```diff
@@ -13,6 +13,8 @@ const RequiredMixin = types
 
     return {
       validate() {
+        const whenRoles = self.whenrole ? self.whenrole.split(",") : null;
```

Comment:
might worth doing `toLowerCase` here and below when you check `includes` so that user won't mess it up